src/components/m-dropdown/_template.js

```diff
@@ -3,17 +3,17 @@ import raw from 'bel/raw';
 
 const arrowIcon = '<axa-icon id="angle-bracket-down" classes="m-dropdown__icon"></axa-icon>';
 
-const nativeSelect = ({ items }) => bel`<div class="m-dropdown__select-wrap" tabindex="0">
-    <select class="m-dropdown__select">
+const nativeSelect = ({ items, size }) => bel`<div class="m-dropdown__select-wrap ${size ? `m-dropdown__select-wrap--${size}` : ''}" tabindex="0">
+    <select class="m-dropdown__select ${size ? `m-dropdown__select--${size}` : ''}">
```

Comment:
This could be even more reduced by using `&&` instead of trinary operator, like:
```js
`${size && `m-dropdown__select-wrap--${size}`}`
```